example_config.toml

```diff
@@ -35,6 +35,9 @@ bootstrap_node = "router.bittorrent.com:6881"
 session = "~/.local/share/synapse/"
 # Default download directory
 directory = "./"
+# Default directory for completed torrents
+# Empty to use the download directory
+completed = ""
```

Comment:
Won't this set completed to an empty string and cause some sort of path error down the line?